src/commands/request/export/readme.ts

```diff
@@ -0,0 +1,71 @@
+import { RequestAction, RequestStatus } from '@transcend-io/privacy-types';
+import { buildExamples } from '../../../lib/docgen/buildExamples';
+import type { ExportCommandFlags } from './impl';
+
+const examples = buildExamples<ExportCommandFlags>(
+  ['request', 'export'],
+  [
+    {
+      description: 'Pull all requests',
+      flags: {
+        auth: '$TRANSCEND_API_KEY',
+      },
+    },
+    {
+      description: 'Filter for specific actions and statuses',
+      flags: {
+        auth: '$TRANSCEND_API_KEY',
+        statuses: [RequestStatus.Compiling, RequestStatus.Enriching],
+        actions: [RequestAction.Access, RequestAction.Erasure],
+      },
+    },
```

Comment:
These are (typesafe) example commands, which get docgen'd into the ```sh block. 
- It will line-break the args if it is >117 chars on a single line (and thus would require horizontal scrolling on github)
- It uses underlying enums (as does the CLI entrypoint/validation), keeping these in sync and documented